CONTRIBUTING.md

```diff
@@ -346,3 +375,31 @@ Even minor versions are dedicated to official releases, e.g. `*.6.*`.
 ### Playground
 
 - [run the playground locally](/website/playground/README.md)
+
+## Releasing
+
+When releasing a new version of a Biome, follow these steps:
+
+1. [ ] Add a [changelog](./CHANGELOG.md) entry for every Pull Request that lacks one.
+   You can filter [merged PRs that don't update the changelog](https://github.com/biomejs/biome/pulls?q=is%3Apr+is%3Amerged+-label%3AA-Changelog).
+   Read our [guidelines for editing the changelog](#changelog).
+
+1. [ ] Based on the [changelog](./CHANGELOG.md), determine which version number to use.
+   See our [versioning guide](https://biomejs.dev/internals/versioning/) for more details.
+
+1. [ ] Rename `Unreleased` to `<version> (iso-date)` in the [changelog](./CHANGELOG.md).
+   Then update the website using `cargo codegen-website`.
+
+1. [ ] Update `version` in [Biome's `package.json`](./packages/@biomejs/biome/package.json).
+
+1. [ ] Update `version` in [Biome's LSP package.json](./editors/vscode/package.json).
+   Note that the LSP follows a [distinct versioning scheme](https://biomejs.dev/internals/versioning/#visual-studio-code-extension).
+
+1. [ ] Linter rules has a `version` metadata directly defined in their implementation.
+   This field is set to `next` for newly added rules.
+   This field must be updated to the new version.
+   Then execute `just codegen-linter`.
+
+1. [ ] Biome's website exposes the JSON schema for `biome.json`.
+   Copy the [up-to-date JSON schema](./packages/@biomejs/biome/configuration_schema.json) to `website/src/pages/schemas/<version>/schema.json.js`.
+   Don't forget to replace `<version>` in the path with the chosen version.
```

Comment:
We can remove this step, it should be covered when we run `cargo codegen-website`